jscomp/others/js_re.mli

```diff
@@ -51,8 +51,12 @@ type t
 (** the result of a executing a RegExp on a string *)
 type result
 
+(** an array of the match and captures, the first is the full match and the remaining are the substring captures *)
+external captures : result -> string Js.nullable array = "%identity"
+
 (** an array of the matches, the first is the full match and the remaining are the substring matches *)
-external matches : result -> string Js.nullable array = "%identity"
+[@@deprecated "Use Js.Re.captures instead"]
```

Comment:
Attributes are usually put *after* the declaration they refer to (Yes, it's weird, but that's OCaml for you ;) ). Mixing before and after can lead to some confusion, so better put it afterwards.